src/stasis/core.clj

```diff
@@ -136,6 +138,10 @@
         (doseq [[uri page] (:dependent-pages page)]
           (export-page uri page target-dir options)))
 
+      (and (nil? page) (:stasis/ignore-nil-pages? options))
+      ;; Do nothing, just return nil
+      nil
```

Comment:
This is also where we would get a bad error message, I think? So again, let's add an exception with some useful debugging info instead. We could even mention the config flag, maybe?